### ISSUE TITLE
Collabora IPv6 check not working

### ISSUE THREAD

**Issue by Zoey2936** [nextcloud/all-in-one#2014]
### Steps to reproduce
1. Use a Domain with IPv6 Support
2. Read logs of Nextcloud container

### Expected behavior 
IPv6 address gets recognized
### Actual behavior 
Config value wopi_allowlist for app richdocuments set to \<ipv4\>,127.0.0.1/8,192.168.0.0/16,172.16.0.0/12,10.0.0.0/8,fd00::/8,::1,\<domain\>

### Host OS 
Debian 11 (Backports)

#### Nextcloud AIO version 
v4.4.1
#### Current channel 
Beta
#### Other valuable info 
```sh
    COLLABORA_IPv6_Address="<?php \$record = dns_get_record('$NC_DOMAIN', DNS_AAAA);"
    COLLABORA_IPv6_Address+='if (!empty($record)) {echo $record[0]["ipv6"];}'
    COLLABORA_IPv6_Address="$(echo "$COLLABORA_IPv6_Address" | php | head -1)"

    if [ -n "$COLLABORA_IPv6_ADDRESS" ]; then
        if ! echo "$COLLABORA_ALLOW_LIST" | grep -q "$COLLABORA_IPv6_ADDRESS"; then
            if [ -z "$COLLABORA_ALLOW_LIST" ]; then
                COLLABORA_ALLOW_LIST="$COLLABORA_IPv6_ADDRESS"
            else
                COLLABORA_ALLOW_LIST+=",$COLLABORA_IPv6_ADDRESS"
            fi
        fi
    else
        echo "No ipv6-address found for $NC_DOMAIN."
    fi
```
```console
+ COLLABORA_IPv6_Address='<?php $record = dns_get_record('\''<domain>'\'', DNS_AAAA);'
+ COLLABORA_IPv6_Address+='if (!empty($record)) {echo $record[0]["ipv6"];}'
++ echo ++ php
'<?php $record = dns_get_record('\''<domain>'\'', DNS_AAAA);if (!empty($record)) {echo $record[0]["ipv6"];}'
++ head -1
+ COLLABORA_IPv6_Address=<ipv6-address>
+ '[' -n '' ']'
+ echo 'No ipv6-address found for <domain>.'
No ipv6-address found for <domain>.
```
```console
bash-5.1# COLLABORA_IPv6_Address="<?php \$record = dns_get_record('$NC_DOMAIN', DNS_AAAA);"
bash-5.1# COLLABORA_IPv6_Address+='if (!empty($record)) {echo $record[0]["ipv6"];}'
bash-5.1# COLLABORA_IPv6_Address="$(echo "$COLLABORA_IPv6_Address" | php | head -1)"
bash-5.1# echo $COLLABORA_IPv6_Address
<ipv6-address>
```
The logic seems to know the IPv6 Address...

**Comment by Zoey2936** [comment]
Maybe because of this:
$COLLABORA_IPv6_**Address**
$COLLABORA_IPv6_**ADDRESS**

**Comment by szaimen** [comment]
> Maybe because of this: $COLLABORA_IPv6_**Address** $COLLABORA_IPv6_**ADDRESS**

Right. Thanks for finding 😅

**Comment by szaimen** [comment]
Do you mind creating a PR? :)

**Comment by Zoey2936** [comment]
> Do you mind creating a PR? :)

done

**Comment by szaimen** [comment]
Thanks! :)

**Comment by szaimen** [comment]
This is now released with v4.5.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
